src/components/RadioBorderedBox/index.tsx

```diff
@@ -27,23 +27,40 @@ const StyledBox = styled(Box)`
   }}
 `
 
-const RadioBorderedBox = ({
+type Props = {
+  badgeSize?: Sizes
+  badgeText?: string
+  badgeVariant?: Variants
+  checked?: boolean
+  children: React.ReactNode
+  disabled?: boolean
+  label: string
+  labelDescription?: string
+  name: string
+  onBlur?(...args: unknown[]): unknown
+  onChange?(...args: unknown[]): unknown
+  onFocus?(...args: unknown[]): unknown
+  size?: number
+  value: string | number
+}
+
+const RadioBorderedBox: FunctionComponent<Props> = ({
   label,
   labelDescription,
   badgeText,
-  badgeSize,
-  badgeVariant,
-  checked,
-  onChange,
-  onFocus,
-  onBlur,
-  disabled,
+  badgeSize = 'xsmall',
+  badgeVariant = 'info',
+  checked = false,
+  onChange = () => undefined,
+  onFocus = () => undefined,
+  onBlur = () => undefined,
+  disabled = false,
   name,
   value,
-  size,
+  size = 24,
   children,
   ...props
-}) => (
+}: Props) => (
```

Comment:
IDK if u need to type here your props because you already did it with `FunctionComponent<Props>`

Reply:
yeah sorry forgot to clean this 👌